aidants_connect_web/models.py

```diff
@@ -381,8 +381,8 @@ def is_active(self):
         # at least one active `autorisation`.
         return self.autorisations.active().exists()
 
-    @property
-    def revocation_date(self):
+    @cached_property
```

Comment:
Tiens c'est marrant, je suis à peu près sûre qu'Anna-Livia avait supprimé ces cached_property à une époque :D 
Pas de souci côté tests ?

Reply:
Elle peut pas avoir supprimé ceux-là parce que c'est des fonctions que j'ai écrites :sweat_smile: 
J'ai pas noté de soucis particuliers sur les tests. J'ai transformé les `@property` en `@cached_property` surtout parce que ça fait des requêtes en BDD un peut lourdes et que j'appelle plusieurs fois ces propriétés, notamment dans les templates. J'avais un peu peur que ça provoque des perfs lamentables.

Reply:
Ahah OK :)